aiostream/aiter_utils.py

```diff
@@ -46,16 +47,38 @@
 # Magic method shorcuts
 
 
+UNSET = object()
```

Comment:
`anext` might be easier to type if `UNSET` is an enum sentinel:
```suggestion
_UnsetType = enum.Enum("_UnsetType", "UNSET")
UNSET = _UnsetType.UNSET
```